ebu_tt_live/scripts/ebu_ebuttd_encoder.py

```diff
@@ -39,6 +39,8 @@
                     default='current', metavar='HH:MM:SS.mmm')
 parser.add_argument('-o', '--output-folder', dest='output_folder', default='./')
 parser.add_argument('-of', '--output-format', dest='output_format', default='xml')
+parser.add_argument('--proxy', dest='proxy', help='Proxy server', type=str, metavar='PROXY:PORT')
```

Comment:
Would be useful to clarify if the protocol prefix is expected here or not, e.g. should it be `http://server_path:port` or just `server_path::port` ?